app/scripts/lib/experiment.js

```diff
@@ -6,8 +6,10 @@ define(function (require, exports, module) {
   'use strict';
 
   const _ = require('underscore');
-  const BaseExperiment = require('lib/experiments/base');
   const ConnectAnotherDeviceExperiment = require('lib/experiments/connect-another-device');
+  // For now, the send SMS experiment only needs to log "enrolled", so
+  // no special experiment is created.
+  const SendSMSExperiment = require('lib/experiments/base');
```

Comment:
I'm not sure about this change, why not just call this `Base` and reuse it if needed if we add more things